code/modules/events/fake_virus.dm

```diff
@@ -6,7 +6,7 @@
 /datum/round_event/fake_virus/start()
 	var/list/fake_virus_victims = list()
 	for(var/mob/living/carbon/human/H in shuffle(GLOB.player_list))
-		if(!H.client || H.stat == DEAD || H.InCritical())
+		if(!H.client || H.stat == DEAD || H.InCritical() || (!SSjob.GetJob(H.mind.assigned_role) || (H.mind.assigned_role in GLOB.nonhuman_positions)))
```

Comment:
This should probably be a macro or a proc if you're using it so frequently.

Reply:
i could stick it in SSJob as a define but I doubt that's necessary